R/util.R

```diff
@@ -23,34 +26,55 @@ ensureBaseUrl <- function() {
   if (pkg.env$baseURL == "") stop("Please call setServer(host) first.")
 }
 
+coerceParameters <- function(params) {
+  logicalToCharacter <- function(v) {
+    if (is.logical(v)) {
+      if (v == TRUE) {
+        return("true")
+      } else {
+        return("false")
+      }
+    }
```

Comment:
`isTRUE`/`isFALSE` would work well here.
```suggestion
    if (isTRUE(v)) {
      return("true")
    } else if (isFALSE(v)) {
      return("false")
    }
```